COC_PyBay.md

```diff
@@ -0,0 +1,76 @@
+#Code of Conduct
+
+PyBay is a regional conference made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching and connecting people.
```

Comment:
Some volunteers *are* professionals. This wording is confusing to me.

Reply:
Could we use these specific terms, if they are necessary? Also, would it make sense to have three categories: "paid conference workers, volunteer conference workers, and conference attendees"? Or possibly drop the categorization and have "PyBay is a regional conference created, organized, and attended by people from all over the world, ..."?

Reply:
done